cognite/seismic/protos/v1/seismic_service_datatypes.proto

```diff
@@ -38,6 +39,34 @@ message OptionalMap {
     map<string, string> data = 1;
 }
 
+// File or dataset or cube derived from a single SEG-Y file
+message CloudStorageFile {
```

Comment:
I have to say I'm not thrilled about this name. The important thing about `File`s is not that they are stored in cloud storage, is it?

What about something like `SourceFile`, or `IngestionFile`, or maybe even `SegyFile`? Something to make it clear that these objects are mostly for ingestion, and it's the `SeismicStore` that's the central data store for querying.

One of the Hard Problems in computer science...

Reply:
i think segyfile or sourcefile is a good idea

Reply:
SourceSegYFile then? =)

Reply:
👍